brion/synapse.cpp

```diff
@@ -137,32 +130,24 @@ class SynapseFile : public boost::noncopyable
         if (!_openDataset(gid, dataset))
             return SynapseMatrix();
 
-        dataset.dataspace.selectNone();
+        std::vector<size_t> columns;
+        columns.reserve(32);
```

Comment:
magic number?

Reply:
Kind of. Being a reserve it doesn't matter very much but I'll use SYNAPSE_ALL instead.